.github/workflows/msys2_clang64-aarch64.yml

```
@@ -0,0 +1,21 @@
+name: MSYS2 aarch64 clang64 build
+on: [push, pull_request, workflow_dispatch]
```

```suggestion
on: [push, pull_request]
```
I don't think we need it - no other job uses it